### PR TITLE
fix: wave target position switch

### PR DESCRIPTION
**Pull request by zombieJ**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

follow up #39954

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |      -     |
| 🇨🇳 Chinese |     和 #39954 一起的，不用写到日志里。      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/version/version.tsx [hunk 1]
```diff
@@ -0,0 +1 @@
+export default '5.1.1';
```
Comment (yoyo837) [inline components/version/version.tsx:1]: 这个文件不是git ignore的?

Comment (zombieJ) [reply components/version/version.tsx:1]: 怪了，怎么会加进去的。我删了

**Review by li-jia-nan — APPROVED** [review]

**Comment by li-jia-nan** [comment]
快速点击改变按钮尺寸的时候好像还是有点问题

```tsx
import React from 'react';
import { Button } from 'antd';

const App: React.FC = () => {
  const [size, setSize] = React.useState<number>(300);
  const onClick = () => {
    setSize(size - 10);
  };
  return (
    <Button style={{ width: size }} onClick={onClick}>
      Button
    </Button>
  );
};

export default App;
```

**Comment by li-jia-nan** [comment]
> 快速点击改变按钮尺寸的时候好像还是有点问题

本来想录个 gif 的，没找到合适的工具……

**Comment by afc163** [comment]
用 quicktime 直接录 video

**Comment by zombieJ** [comment]
> 快速点击改变按钮尺寸的时候好像还是有点问题

嗯，尺寸和位置变化暂时不考虑加 Observer。这种场景毕竟还是比较少。有真实 Case 的时候再加也不迟哈~

**Comment by li-jia-nan** [comment]
嗯，我是看到了官网带 loading 的例子，点击之后进入 loading 状态，然后出现 loading icon，按钮尺寸发生了变化，所以想到了这个

**Comment by zombieJ** [comment]
> 嗯，我是看到了官网带 loading 的例子，点击之后进入 loading 状态，然后出现 loading icon，按钮尺寸发生了变化，所以想到了这个

合理，加之~

**Pull request merged**